python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -231,9 +231,13 @@ cpdef read_parquet(filepaths_or_buffers, columns=None, row_groups=None,
                 column_names.remove(index_col)
 
         for col in column_names:
+            try:
+                data = cols_dtype_map[col]
+            except KeyError:
+                data = cols_dtype_map.get(col, None)
```

Comment:
Instead of using a `try except` here can we just always use `cols_dtype_map.get(col, None)`?

Reply:
Also would suggest using a more descriptive variable name than `data`

Reply:
Sounds good! I'll make the updates for this